### PR TITLE
WIP Test load ->dump ->load

### PR DESCRIPTION
**Pull request by kidd**
Integrate test-load-and-dump into the regular test suite

### CODE REVIEW

File: test/metabase/db/load_and_dump_test.clj [hunk 1]
```diff
@@ -0,0 +1,51 @@
+(ns metabase.db.load-and-dump-test
```
Comment (camsaul) [inline test/metabase/db/load_and_dump_test.clj:1]: Test namespaces should be `ORIGINAL-NAMESPACE-test` e.g. `metabase.cmd.load-from-h2-test`. Tools like CIDER know that namespaces are associated with one another. 

For places that don't really have a corresponding match you should still try to put them in the same folder so I would name this `metabase.cmd.load-and-dump-test`

File: test/metabase/db/load_and_dump_test.clj [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+(deftest load-and-dump-test
+  (testing "loading data to h2 and porting to DB and migrating back to H2"
+    ;; do we import clojure.java.io even for just one call?
+    (let [h2-fixture-db-file (abs-path "frontend/test/__runner__/test_db_fixture.db")
+          h2-file (abs-path "frontend/test/__runner__/test_db_fixture2.db")
+          db-name "dump-test"]
+
+      ;; XXX how the :mysql works if I don't have it running?
+      (mt/test-drivers #{:postgres :mysql}
+
```
Comment (camsaul) [inline test/metabase/db/load_and_dump_test.clj:32]: https://guide.clojure.style/#no-blank-lines-within-def-forms

I think there are a few exceptions:
- breaking out a giant `deftest` form into a few different `testing` sections
- breaking out different arities of a `defn` form (this is used as a "bad" example but I don't think it's that bad)

either way, there shouldn't be any blank lines here

File: test/metabase/cmd/load_and_dump_test.clj [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            [toucan.db :as db]))
+
+(defn- abs-path
+  [path]
+  (.getAbsolutePath (io/file path)))
+
+(deftest load-and-dump-test
+  (testing "loading data to h2 and porting to DB and migrating back to H2"
+    (let [h2-fixture-db-file (abs-path "frontend/test/__runner__/test_db_fixture.db")
+          h2-file (abs-path "frontend/test/__runner__/test_db_fixture2.db")
```
Comment (camsaul) [inline test/metabase/cmd/load_and_dump_test.clj:25]: for the destination file I'd just put it in `/tmp` so if you run this locally you don't end up accidentally committing it

Comment (kidd) [reply test/metabase/cmd/load_and_dump_test.clj:25]: ah, great. it was my first approach but then thought that maybe in circle-ci the `/tmp/...` paths would collide if different jobs run in parallel.  But it seems it's fine, so I'll change it.

**Review by camsaul — APPROVED** [review]
Cool, this looks good to me 👍

**Pull request closed without merging**